modules/microshift-nw-multus-add-pod.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microhift-cni-multus.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-multus-add-pod_{context}"]
+= Adding a pod to an additional network
+
+You can add a pod to an additional network. The pod continues to send normal cluster-related network traffic over the default network.
+
+When a pod is created additional networks are attached to it. However, if a pod already exists, you cannot attach additional networks to it.
+
+The pod must be in the same namespace as the additional network.
+
+.Prerequisites
+
+* The OpenShift CLI (`oc`) is installed.
+* The cluster is running.
+* A network defined by a `NetworkAttachmentDefinition` object that you want to attach the pod to exists.
+
+.Procedure
+
+. Add an annotation to the `Pod` object. Only one of the following annotation formats can be used:
+
+.. To attach an additional network without any customization, add an annotation with the following format. Replace `<network>` with the name of the additional network to associate with the pod:
++
+[source,yaml]
+----
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: <network>[,<network>,...] <1>
+----
+<1> To specify more than one additional network, separate each network with a comma. Do not include whitespace between the comma. If you specify the same additional network multiple times, that pod will have multiple network interfaces attached to that network.
+
+.. To attach an additional network with customizations, add an annotation with the following format:
```

Comment:
So this something I didn't verify. I wasn't even aware this exists and looking now at the code, this config could be much more comprehensive. I see tests for this in multus repository so I guess we can keep it